tictactoe.py

```diff
@@ -127,7 +127,7 @@ def run(self):
 			else:
 				print("Position Not Empty!!")
 
-
+print("Welcome TO Tic Tac Toe")
```

Comment:
Why is "TO" all caps? Please make it "To".

Reply:
corrected